src/libutil/widecharwidth/widechar_width.h

```diff
@@ -1531,7 +1531,7 @@ bool widechar_in_table(const Collection &arr, uint32_t c) {
 }
 
 /* Return the width of character c, or a special negative value. */
-int widechar_wcwidth(uint32_t c) {
+inline int widechar_wcwidth(uint32_t c) {
```

Comment:
nit
```suggestion
inline int widechar_wcwidth(uint32_t c) 
{
```

Reply:
This code is vendored. We should probably not change the code style.